src/Data/ProgrammesDb/Entity/EntityInterfaces/RelatedInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Data\ProgrammesDb\Entity\EntityInterfaces;
+
+interface RelatedInterface
```

Comment:
Pondering on some better names for these interfaces: RelatedLinkContextInterface?